aspnetcore/fundamentals/host/generic-host/samples/2.x/GenericHostSample/NuGet.config

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <packageSources>
+    <clear />
+    <add key="dotnet-core" value="https://dotnet.myget.org/F/dotnet-core/api/v3/index.json" />
```

Comment:
Create an issue, if you haven't already, to remove the MyGet package source once the associated packages are published to NuGet.

Reply:
Already done :+1: It's listed in the sample tracking issue for 2.1 updates. #5495